data/scripts/apricorn_tree.inc

```diff
@@ -0,0 +1,85 @@
+ApricornTreeScript::
+	message ApricornTree_Text_Intro
+	special ObjectEventInteractionGetApricornTreeData
+	goto_if_gt VAR_0x8005, 0, ApricornTree_EventScript_WantToPick
+	message ApricornTree_Text_Empty
+	end
+
+ApricornTree_EventScript_WantToPick::
+	buffernumberstring STR_VAR_2, VAR_0x8005
+	lock
+	faceplayer
+	msgbox ApricornTree_Text_WantToPick, MSGBOX_YESNO
+	goto_if_eq VAR_RESULT, YES, ApricornTree_EventScript_PickApricorn
+	goto_if_eq VAR_RESULT, NO, ApricornTree_EventScript_CancelPickingApricorn
+
+.set APRICORN_NORMAL_BAG_FULL, 0
+.set APRICORN_NORMAL_SPACE_IN_BAG, 1
+
+ApricornTree_EventScript_PickApricorn::
+	special ObjectEventInteractionPickApricornTree
+	call EventScript_BufferPocketNameAndTryFanfare
+	goto_if_eq VAR_0x8006, APRICORN_NORMAL_BAG_FULL, ApricornTree_EventScript_PocketFull
+	message ApricornTree_Text_PickedTheApricorn
+	.if OW_SHOW_ITEM_DESCRIPTIONS != OW_ITEM_DESCRIPTIONS_OFF
```

Comment:
```suggestion
.if OW_SHOW_ITEM_DESCRIPTIONS != OW_ITEM_DESCRIPTIONS_OFF
```

Configs in .inc scripts are indented to the left, also applies to the rest of the file